packages/dlp/src/index.js

```diff
@@ -37,13 +39,25 @@ const VERSION = require('../package.json').version;
  * The service also includes methods for sensitive data redaction and scheduling
  * of data scans on Google Cloud Platform based data sets.
  *
+ * The servicePath from options will set the host. If not set, the
+ * `GOOGLE_CLOUD_DLP_ENDPOINT` environment variable is honored,
+ * otherwise the actual API endpoint will be used.
+ *
  * @param {object=} options - [Configuration object](#/docs).
  * @param {number=} options.port - The port on which to connect to
  *     the remote host.
  * @param {string=} options.servicePath - The domain name of the
  *     API remote host.
  */
 function dlpV2beta1(options) {
+  options = common.util.resolveGapicOptions(
```

Comment:
Editing these files will be a bit of a problem, as they are generated. @landrito -- is it possible to have the generator start spitting out these new lines? This PR adds support for custom endpoints, so a user can talk to an emulator or other arbitrary endpoint.

Reply:
Hm. I'd rather not pull in @google-cloud/common as a dependency to the generated clients. @lukesneeringer this might be where we push for merging @google-cloud/common and google-gax into a single package like python did.

Reply:
Ping!